boilerplate/test/test-base-convention/standard.mk

```diff
@@ -37,7 +37,7 @@ GOENV=GOOS=${GOOS} GOARCH=${GOARCH} CGO_ENABLED=0 GOFLAGS=
 
 GOBUILDFLAGS=-gcflags="all=-trimpath=${GOPATH}" -asmflags="all=-trimpath=${GOPATH}"
 
-TESTTARGETS := $(shell ${GOENV} go list -e ./... | egrep -v "/(vendor)/")
+TESTTARGETS := $(shell ${GOENV} go list -e ./... | egrep -v "/(vendor)/" | grep -v /int)
```

Comment:
Couple things here:
- I'd like to get a bit more precise with the regex, as this will match any package anywhere whose name starts with `int`. Something like `egrep -v '/int(/.*)?$'` could work ("/int" followed by an optional subdirectory structure, then explicit EOL). 
- Can we please add a comment explaining why we're excluding this? I feel like `/vendor/` is fairly obvious, but this isn't.

All that said, I'm not loving that we're reserving "int", which seems like it could show up in any number of packages we legitimately want to test. It would be tough to notice that we're missing testing it, and then it would be awkward to resolve, especially if time had passed and we had already published e.g. `pkg/apis/int/v1`. If it were narrower, like `test/integration` (`/test/integration(/.*)?$`), I would feel more comfortable with it. (Obviously that would entail RMO re-pathing these tests. I would think that's not a show-stopper at this point.)

Another strategy is to override `TESTTARGETS` in your specific project. As written, you could simply assign it after the boilerplate `include`. Or we could change `:=` to `?=` here and the consumer could override it before or after. (The latter is probably better, in case we later add some other variable that derives from `TESTTARGETS` at the top level.)

Regardless, whatever we decide to do to boilerplate's make vars/rules will need to be done in boilerplate/openshift/golang_osd_operator/standard.mk. This file is only used for the internal test suites.

Reply:
whoops selected the wrong file when I reapplied my changes. Yes, a way to override it would be a nicer I think. Will change the PR.